src/styles/blocks/second.scss

```diff
@@ -0,0 +1,23 @@
+.second {
+  width: 4px;
+  height: 200px;
```

Comment:
this work is for a CV, here you can separate (size and color ) it with a space )))